### ISSUE TITLE
Problem: Poll results wrongly shown when using more than 4 options

### ISSUE THREAD

**Issue by jorgesat** [magicstone-dev/ecko#377]
When using more than 4 options in polls, the vote count is correct but the resulting report, when the poll has finished, shows the check-mark ✔️ (your own vote) in a random option (not the one you voted).

**Comment by Mansh05** [comment]
Thanks for the report @jorgesat, Could you please show one screen shot as well. I will check the UI and see the issue.

**Comment by weex** [comment]
Maybe related to randomization of poll options done with https://github.com/magicstone-dev/ecko/pull/243

**Comment by weex** [comment]
Confirmed on a vote I participated in. I chose the fifth option out of seven shown to me and when I reload, Ecko randomizes the options but reports that I voted for the fifth option, whatever that might be. Possible solution is revert #243 since it was a pretty simple one and also more of an early experiment than a feature anyone in the community asked for.

<img width="570" alt="Screen Shot 2022-01-13 at 5 29 44 PM" src="https://user-images.githubusercontent.com/38776/149435704-eaaa262b-c22e-4e74-bad6-0a4462f47907.png">
